enterprise/server/cmd/ci_runner/main.go

```diff
@@ -1929,6 +1930,12 @@ func runCommand(ctx context.Context, executable string, args []string, env map[s
 	if dir != "" {
 		cmd.Dir = dir
 	}
+
+	if !*interactive {
```

Comment:
Probably should be `!*interactive  && *workflowID == ""` so we continue to use a PTY for workflows, since BB workflows supports ANSI sequences.